product-bundles-variation-bundles.php

```diff
@@ -621,6 +621,13 @@ public static function variation_get_shipping_class_id( $shipping_class_id, $var
 	 */
 	public static function variation_get_tax_class( $tax_class, $variation ) {
 
+		$parent_id      = $variation->get_parent_id();
+		$parent_product = wc_get_product( $parent_id );
+
+		if ( 'none' === $parent_product->get_tax_status() ) {
```

Comment:
Wondering if it would be possible to retrieve this via `$variation->get_parent_data()`?

Reply:
I had the same thought too but `tax_status` isn't a key in the parent data array.

Reply:
I'm a bit concerned about performance here, wondering if we could wrap `wc_get_product` inside a `get_variation_parent` that runtime-caches the result?